### DISCUSSION TITLE
Newbie tries to fix something gpt-pilot can't fix

### DISCUSSION THREAD

**Discussion by charliecalvert** [Pythagora-io/gpt-pilot#88] [Q&A] [answered]
Hopefully I am asking for a solution to a question. If not, then I am asking for a new feature. Please forgive my extreme newbie status.

I tried to get GPT pilot to create a login script for Postgres. I'm probably asking the question wrong, but regardless, I found the solution. I put it into the main file generated by gpt-pilot. It works. 

Now, how do I tell the gpt-pilot to move on to the next step based on the existing code. It keeps deleting what I've included in the main file and starting over and we get nowhere. Is there a way to say don't delete this?

I don't want to know the correct way to ask the question, I want to know how to tell gpt-pilot to keep moving from the current state.

Maybe another way to ask the question: "How can I ask gpt-pilot to add a feature to an existing program or code base?  Start here and add this feature..."

**Comment by LeonOstrez** [comment, marked as answer]
Currently we don't have a feature to analyse already existing project and build on top of it (new features or updates/fixes to existing code). We have researched that area and we will definitely have that in future.

For now gpt-pilot can only add new features and modifications to projects that was built by gpt-pilot.
